platform/mellanox/hw-management.mk

```diff
@@ -1,6 +1,6 @@
 # Mellanox HW Management
 
-MLNX_HW_MANAGEMENT_VERSION = V1.0.0165
+MLNX_HW_MANAGEMENT_VERSION = V.1.0.0166
```

Comment:
>. [](start = 30, length = 1)

Why the version string is weird?

Reply:
@qiluo-msft This is a new tag which was provided by HW-MGMT: https://github.com/Mellanox/hw-mgmt/tree/V.1.0.0166